File: Code/c/number_length.c
#include<stdio.h>
void main()
{
    int num, length = 1;

    printf("Enter Number: ");
    scanf("%d", &num);

    while (num / 10 != 0)
    {
        num /= 10;
        length++;
    }
    printf("%d", length);
}

File: Code/java/number_length.java
import java.util.Scanner;

class NumberLength
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Number: ");
        System.out.println(Integer.toString(scanner.nextInt()).length());

        scanner.close();
    }
}